roles/pulibrary.pulmap/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+galaxy_info:
+  author: Princeton University Library
+  description: pulmap role
+
+  license: MIT
+
+  min_ansible_version: 2.8
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - 18.04
+dependencies:
```

Comment:
Are the `deploy_user`, `passenger`, and `nodejs` roles no longer needed?

Reply:
the rails_app (listed as dependency will bring those in)